src/overlays/actors/ovl_En_Aob_01/z_en_aob_01.c

```diff
@@ -60,13 +61,13 @@ typedef enum {
     /* 3 */ EN_AOB01_EYE_MAX
 } EnAob01EyeTexture;
 
-static AnimationInfo sAnimationInfo[] = {
-    { &gMamamuYanIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },
-    { &gMamamuYanLaughStartAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, 0.0f },
-    { &gMamamuYanLaughLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },
-    { &gMamamuYanSurpriseStartAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, 0.0f },
-    { &gMamamuYanSurpriseLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },
-    { &gMamamuYanIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -6.0f },
+static AnimationInfo sAnimationInfo[EN_AOB01_ANIM_IDLE_MAX] = {
+    { &gMamamuYanIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },          // EN_AOB01_ANIM_IDLE_1
```

Comment:
Wasn't you, but the difference between this and the other one is just the morph frames. Perhaps drop the 1 here and name the other `EN_AOB01_ANIM_IDLE_MORPH`?

Reply:
Yup, makes sense to me

Reply:
Makes sense to me too, and I'm the one that documented this actor

Reply:
Although `EN_AOB01_ANIM_IDLE_MAX` should be `EN_AOB01_ANIM_MAX`, oops. Fixed now